### PR TITLE
Create and activate AWS 9.0.3 release with containerlinux 2247.7.0

### PR DESCRIPTION
**Pull request by tfussell**

### CODE REVIEW

**Comment by cokiengchiara** [comment]
hi @tfussell - a request, can you help with communicating this release? :)

1. Create a notecard in Releases Board as soon as your team decides you want and plan to make a new release
2. Write a short announcement in news-product channel with a link to the notecard / issue in Releases Board. 

This makes other teams aware that you are creating a release. This helps us find inter-team dependencies quickly and, where possible, try to avoid them.

See https://github.com/giantswarm/giantswarm/blob/master/content/docs/product/release-coordination.md#process for details and scripts :)

File: aws.yaml [hunk 1]
```diff
@@ -551,9 +551,61 @@ kind: Release
...
-  name: v9.0.2
+  name: v9.0.3
 spec:
   state: active
+  date: 2020-04-17T12:00:00Z
+  apps:
+  - name: cert-exporter
+    version: 1.2.1
+  - name: chart-operator
+    version: 0.12.1
```
Comment (cokiengchiara) [inline aws.yaml:562]: Also, would like latest chart-operator 0.12.4 in here. https://github.com/giantswarm/chart-operator/blob/master/CHANGELOG.md#v0124-2020-04-15 

cc @rossf7 please confirm if correct :)

**Comment by tfussell** [comment]
@cokiengchiara Certainly! Thanks for laying it out. I'm testing this on the technical side first before I decide to go with this. I'll be sure to follow those steps if this seems like the right approach.

File: aws.yaml (hunk 1 above)
Comment (rossf7) [reply aws.yaml:562]: Yes please update include it. It has several fixes missing in 0.12.1.

**Comment by sslavic** [comment]
@giantswarm/team-ludacris @tfussell  what's the ETA on this? If it's not going to ship this week, would you mind if this gets changed to 9.0.4, and batman and halo to release 9.0.3 with chart-operator and nginx app update only?

**Comment by tfussell** [comment]
> @giantswarm/team-ludacris @tfussell what's the ETA on this? If it's not going to ship this week, would you mind if this gets changed to 9.0.4, and batman and halo to release 9.0.3 with chart-operator and nginx app update only?

My test didn't work. It will take some extra changes to the operator so we're focusing on 9.3.0 with versions from releases and Flatcar this week. Feel free to take it over, or I can close it in favor of yours.

**Comment by sslavic** [comment]
@tfussell lets close this one, I'll create a new one with app updates only.

**Pull request closed without merging**